templates/server-statefulset.yaml

```diff
@@ -197,6 +197,13 @@ spec:
                 -hcl='ports { http = -1 }' \
                 {{- end }}
                 {{- end }}
+                {{- if .Values.server.partitions }}
+                {{- $partitions := "" }}
+                {{- range $partition := .Values.server.partitions }}
+                {{- $partitions = printf "%s{name = \\\"%s\\\", bind = \\\"0.0.0.0\\\", advertise = \\\"%s\\\", port = %v}," $partitions $partition.name "${ADVERTISE_IP}" $partition.port -}}
```

Comment:
are the \'s for making newlines and tabs? just curious :)

Reply:
:) once the values here are filled in it translates into one single line which has comma separated entries. 
-hcl="segments = [{name = "foo", bind="0.0.0.0", advertise = "1.2.3.4", port = 2000},{name = "bar", advertise = "0.0.0.0", advertise = "2.3.4.5", port = 2020},]" etc
does this answer the question?